StarkInfra/StarkInfra/Event/Attempt.cs

```diff
@@ -97,8 +97,8 @@ public static Attempt Get(string id, User user = null)
             ///     <item>limit [integer, default null]: maximum number of objects to be retrieved. Unlimited if null. ex: 35</item>
             ///     <item>after [DateTime, default null] date filter for objects created only after specified date. ex: DateTime(2020, 3, 10)</item>
             ///     <item>before [DateTime, default null] date filter for objects created only before specified date. ex: DateTime(2020, 3, 10)</item>
-            ///     <item>eventIds [list of strings, default null]: list of Event ids to filter attempts. ex: ["5656565656565656", "4545454545454545"]</item>
-            ///     <item>webhookIds [list of strings, default null]: list of Webhook ids to filter attempts. ex: ["5656565656565656", "4545454545454545"]</item>
+            ///     <item>eventIds [list of strings, default null]: list of Event ids to filter attempts. ex: new List<string>{ "5656565656565656", "4545454545454545" }</item>
+            ///     <item>webhookIds [list of strings, default null]: list of Webhook ids to filter attempts. ex: new List<string>{ "5656565656565656", "4545454545454545" }</item>
             ///     <item>user [Project object, default null]: Project object. Not necessary if StarkInfra.User.Default was set before function call</item>
```

Comment:
Aqui no user deveria ser: `[Organization/Project object, default None]: Organization or Project object. Not necessary if Starkinfra.User.Default was set before function call.`

Vi que tem uns lugares que estão assim também, da um search all pra ajustar tudo.